core/src/main/java/tc/oc/pgm/spawner/objects/SpawnableItem.java

```diff
@@ -11,18 +11,18 @@
 
 public class SpawnableItem implements Spawnable {
 
-  private ItemStack stack;
-  private String METADATA_VALUE;
+  private final ItemStack stack;
+  private final String metadataValue;
 
-  public SpawnableItem(ItemStack stack, int spawnerID) {
+  public SpawnableItem(ItemStack stack, int spawnerId) {
```

Comment:
```suggestion
  public SpawnableItem(ItemStack stack, String spawnerId) {
```
Pass in a string directly, and use the actual id of the spawner specified in xml if one is available (otherwise, use an autogenerated one like "spanwer-0", "spanwer-1" etc)